LetsDoStuff.WebApi/Controllers/ParticipationRequestorController.cs

```diff
@@ -9,14 +9,33 @@
 
 namespace LetsDoStuff.WebApi.Controllers
 {
-    [Route("api/participation")]
-    public class ParticipationController : ControllerBase
+    [Route("api/ParticipationRequestor")]
+    public class ParticipationRequestorController : ControllerBase
     {
-        private readonly IParticipationService _participationService;
+        private readonly IParticipationRequestorService _participationRequester;
```

Comment:
Why do you think the name is better?

Reply:
It was renamed by default, so there is no reason why this name is better or worse.